test/Interop/Cxx/ergonomics/Inputs/implicit-computed-properties.h

```diff
@@ -0,0 +1,189 @@
+#ifndef SWIFT_IMPLICIT_COMPUTED_PROPERTIES_H
+#define SWIFT_IMPLICIT_COMPUTED_PROPERTIES_H
```

Comment:
Everything here needs to have consistent formatting that matches the rest of the project (sans the names obviously).

Reply:
I'm really just saying indent everything with two spaces.